spec/features/proposals/index_spec.rb

```diff
@@ -20,20 +20,72 @@
     user = create(:user)
     proposals = create_list(:proposal, 2, observer: user)
     cancelled = create_list(:proposal, 2, status: "cancelled", observer: user)
+    @page = MyRequestsPage.new
```

Comment:
what about naming it `ProposalsIndexPage` ? seems like that might be an easier convention moving forward (name after controller / action)